src/common/models/app-settings/app-settings.ts

```diff
@@ -125,8 +125,13 @@ export class AppSettings implements Instance<AppSettingsValue, AppSettingsJS> {
 
   public toClientSettings(): AppSettings {
     var value = this.valueOf();
+
     value.clusters = value.clusters.map((c) => c.toClientCluster());
-    value.dataSources = value.dataSources.map((ds) => ds.toClientDataSource());
+
+    value.dataSources = value.dataSources
+      .filter((ds) => ds.isQueryable())
```

Comment:
Okay this shouldn't have passed the tests : running them locally shows one failing tests on app-settings, directly linked to this (data-source mocks don't have an executor and therefore aren't queryable, which messes up the app-settings tests)... How come Travis didn't notice anything ? Come on Travis, you're embarrassing yourself.